site/docs/edgent-getting-started.md

```diff
@@ -25,32 +25,43 @@ Edgent's primary API is functional where streams are sourced, transformed, analy
 
 ### Downloading Apache Edgent
 
-To use Edgent you need the Edgent jars. You can either download the Edgent source and build it or you can download a pre-built version of Edgent. See the [download page]({{ site.data.project.download }}).
-
-After you have the Edgent jars, you can set up your environment.
+To use Edgent, you need the Edgent JAR files, which you may obtain by completing the following steps.
+
+1. Locate the Edgent release you would like to use on the [downloads page]({{ site.data.project.download }})
+2. In the Bundles column for the desired release:
+    * Click on the _Binary_ link if you simply want to use Edgent. This is the easiest method is to get up and running as it contains a pre-built version of Edgent.
+    * Click on the _Source_ link if you would like access to the Edgent source files. If you choose this method, you must manually build Edgent yourself.
+3. Download the .tgz file from one of the mirror sites
+4. Unpack the downloaded file: `tar zxvf apache-edgent-X.X.X-incubating-XXX.tgz`
+5. Obtain the JARs
+    * If you are using a binary bundle, then the Java 8 JARs are located in `edgent-X.X.X/java8`
+    * If you are using a source bundle, build the source code:
+        1. Install [Gradle](https://gradle.org/) if it is not yet installed
+        2. Navigate to the unpacked directory: `cd edgent-X.X.X-src`
+        3. Run `gradle` to initialize the Gradle wrapper
+        4. Build the code and Javadoc: `./gradlew assemble`
+        5. The Java 8 JARs are located in `edgent-X.X.X-src/build/distributions/java8`
 
 ### Setting up your environment
 
-Ensure that you are running a supported environment. For more information, see the [Edgent overview](home). This guide assumes you're running Java 8.
-
-The Edgent Java 8 JAR files are located in the `edgent/java8/lib` directory.
+Ensure that you are running a supported environment. For more information, see the [Edgent overview](home). This guide assumes you're running Java 8. The Edgent Java 8 JAR files are located in either the `edgent-X.X.X/java8` or `edgent-X.X.X-src/build/distributions/java8` directory, depending on whether you downloaded a binary or source bundle.
 
 1. Create a new Java project in Eclipse, and specify Java 8 as the execution environment JRE:
 
-    <img src="images/New_Java_Project.JPG" style="width:448px;height:589px;">
+    <img src="images/New_Java_Project.jpg">
 
 
-2. Modify the Java build path to include all of the JAR files in the `edgent\java8\lib` directory:
+2. Add all of the JAR files in the `java8\lib` directory to the project's build path:
```

Comment:
An SLF4J implementation jar also needs to be included (e.g., java8/ext/slf4j-jdk14-X.X.X.jar) to avoid SLF4J warnings to stderr, right?

Might it help users to note the build path also needs to include the jars for any Edgent connector or analytic component used by an application (under java8/connectors and java8/analytics respectively)?  This getting-started sample doesn't require any.

Fwiw, the only lib jar needed is the one for the provider used by the application.  Our jars' manifest adds in all dependent lib jars.  See [Build Applications](https://edgent.apache.org/javadoc/latest/index.html) at the bottom of its page.  So for this sample only 2 jars are actually needed: the ext/slf-jdk jar and lib/edgent.providers.direct.jar.  Including all of the provider jars should cover any app.  It might be nice to see the trimmed down list in the dialog but I understand that saying "include all lib jars" may be the easiest. Maybe something to consider.